scripts/cleanup_edomain_templates.py

```diff
@@ -100,25 +102,35 @@ def get_edomain_path(api, edomain):
             edomain_conn.get_address())
 
 
-def cleanup_empty_dir_on_edomain(path, edomainip, sshname, sshpass):
+def cleanup_empty_dir_on_edomain(path, edomainip, sshname, sshpass, provider_ip, provider):
```

Comment:
@psav  In the new commit changed that to pass the provider_obj and fetch the provider_ip and provider_name from that.